### PR TITLE
Update readme docsng 749

### PR DESCRIPTION
**Pull request by techwritermat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,152 @@
-# python-quickstart-platform
+# Python PubNub Sample App
```
Comment (polarweasel) [inline README.md]:
```suggestion
# PubNub Python Sample App
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,152 @@
...
+
+        if envelope.status.is_error():
+            print("[PUBLISH: fail]")
+            print("error: %s" % envelope.status.error)
+            pass
+        else:
+            print("[PUBLISH: sent]")
+            print("timetoken: %s" % envelope.result.timetoken)
+            pass
+
```
Comment (polarweasel) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,152 @@
...
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
+   
+1. Create a new folder for your Python scripts.
+
+2. In that folder, create two files named `pub.py` and `sub.py`.
+
+3. Install the PubNub Python SDK using `pip`:
+
+    ```text
+    pip install 'pubnub>=4.5.1'
```
Comment (polarweasel) [inline README.md]: 4.6.0 dropped yesterday :)

```suggestion
    pip install 'pubnub>=4.6.0'
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,152 @@
...
+    ```
+
+## Run the app
+
+You first need to start the subcribe process to see the updates you want to publish.
+
+1. Open the terminal and run the subscribe script by calling `python sub.py`.
+
+    You should see a message that the script is waiting for updates. The subscriber process is implemented to receive updates from the publisher process, so you don't need any further input here.
+
```
Comment (polarweasel) [inline README.md:126]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,152 @@
...
+    When you're finished submitting entries, you can enter `42` to exit the publisher script. Note that this operation forces the subscriber script to exit.
+
+## Documentation
+
+* [Build your first realtime Python app with PubNub](https://www.pubnub.com/docs/platform/quickstarts/python)
+* [API reference for Python](https://www.pubnub.com/docs/python/pubnub-python-sdk)
+
+## Support
+
+If you **need help** or have a **general question**, contact <support@pubnub.com>.
```
Comment (polarweasel) [inline README.md]:
```suggestion
If you **need help** or have a **general question**, contact support@pubnub.com.
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,152 @@
...
+1. Open the terminal and run the subscribe script by calling `python sub.py`.
+
+    You should see a message that the script is waiting for updates. The subscriber process is implemented to receive updates from the publisher process, so you don't need any further input here.
+
+
+2. In a new terminal window, run the publish script by calling `python pub.py`.
+
+    You should see a message that informs you how to exit the process and a prompt that awaits your next update.
+
+    When you're finished submitting entries, you can enter `42` to exit the publisher script. Note that this operation forces the subscriber script to exit.
```
Comment (polarweasel) [inline README.md]:
```suggestion
    When you're finished submitting entries, enter `42` to exit the publisher script. This operation also forces the subscriber script to exit.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Looks good! Minor changes only here

File: README.md [hunk 7]
```diff
@@ -1,3 +1,150 @@
...
+    while the_update != "42":
+        print
+        the_update = input("Enter an update for Earth: ")
+        the_message = {"entry": ENTRY, "update": the_update}
+        envelope = pubnub.publish().channel(CHANNEL).message(the_message).sync()
+
+        if envelope.status.is_error():
+            print("[PUBLISH: fail]")
+            print("error: %s" % envelope.status.error)
+            pass
```
Comment (650elx) [inline README.md]: Please remove all redundant "pass" statements.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,150 @@
...
+
+    pubnub = PubNub(pnconfig)
+
+    class MySubscribeCallback(SubscribeCallback):
+        def message(self, pubnub, event):
+            print("[MESSAGE received]")
+
+            if event.message["update"] == "42":
+                print("The publisher has ended the session.")
+                os._exit(0)
```
Comment (650elx) [inline README.md]: Do we have to use os_exit(0) here?

File: README.md [hunk 9]
```diff
@@ -1,3 +1,150 @@
...
+                print("[STATUS: PNConnectedCategory]")
+                print("connected to channels: {}".format(event.affected_channels))
+                print
+            
+            pass
+
+    pubnub.add_listener(MySubscribeCallback())
+    pubnub.subscribe().channels(CHANNEL).with_presence().execute()
+
+    print
```
Comment (650elx) [inline README.md]: Redundant "print" statements.

Comment (650elx) [reply README.md]:
```
>>> print
<built-in function print>
>>> print()

>>> 
```
If you're looking for a newline... :)

**Review by 650elx — CHANGES_REQUESTED** [review]

**Review by polarweasel — DISMISSED** [review]
Looks good, pending Bartosz's changes

File: README.md (hunk 7 above)
Comment (techwritermat) [reply README.md]: done

File: README.md (hunk 9 above)
Comment (techwritermat) [reply README.md]: thanks!

File: README.md (hunk 8 above)
Comment (techwritermat) [reply README.md]: I think so, the script should close both scripts `pub` and `sub`.

**Review by 650elx — DISMISSED** [review]

**Review by crimsonred — APPROVED** [review]

**Pull request merged**